data/creaturescripts/scripts/report.lua

```diff
@@ -0,0 +1,26 @@
+function onReport(player, message, position, category)
+	if player:getAccountType() == ACCOUNT_TYPE_NORMAL then
+		return false
+	end
+
+	local name, playerPos = player:getName(), player:getPosition()
```

Comment:
Also in the update, split this variable, since if the file does not exist there is no reason to fetch the player position.
Also please use playerPosition or just position insted of playerPos :)